static/app/views/performance/landing/widgets/widgets/performanceScoreListWidget.tsx

```diff
@@ -0,0 +1,282 @@
+              </GrowLink>
+              <StyledRightAlignedCell>
+                {listItem.score !== null && (
+                  <Tooltip
+                    title={
+                      <span>
+                        {t('The overall performance rating of this page.')}
+                        <br />
+                        <ExternalLink href="https://docs.sentry.io/product/performance/web-vitals/#performance-score">
+                          {t('How is this calculated?')}
+                        </ExternalLink>
+                      </span>
+                    }
+                    isHoverable
+                  >
+                    <PerformanceBadgeWrapper>
+                      <PerformanceBadge score={listItem.score} />
+                    </PerformanceBadgeWrapper>
+                  </Tooltip>
+                )}
+                {isProjectWebVitalDataLoading ||
+                (isProjectScoresLoading && shouldUseStoredScores) ? (
+                  <StyledLoadingIndicator size={20} />
+                ) : (
+                  <Tooltip
+                    title={
+                      <span>
+                        {t(
+                          "A number rating how impactful a performance improvement on this page would be to your application's overall Performance Score."
+                        )}
+                        <br />
+                        <ExternalLink href="https://docs.sentry.io/product/performance/web-vitals/#opportunity">
+                          {t('How is this calculated?')}
+                        </ExternalLink>
+                      </span>
+                    }
+                    isHoverable
+                    showUnderline
+                    skipWrapper
+                  >
+                    {Math.round(opportunity * 100) / 100}
+                  </Tooltip>
+                )}
+              </StyledRightAlignedCell>
+            </Fragment>
+          );
+        }
+    );
+
+  const getContainerActions = _ => {
+    return (
+      <Fragment>
+        <div>
+          <LinkButton
+            to={`/organizations/${organization.slug}/performance/browser/pageloads/`}
+            size="sm"
+          >
+            {t('View All')}
+          </LinkButton>
+        </div>
+        {ContainerActions && (
+          <ContainerActions isLoading={isTransactionWebVitalsQueryLoading} />
+        )}
+      </Fragment>
+    );
+  };
+
+  return (
+    <GenericPerformanceWidget<DataType>
+      {...props}
+      location={location}
+      Subtitle={() => <Subtitle>{props.subTitle}</Subtitle>}
+      HeaderActions={provided => getContainerActions(provided)}
+      InteractiveTitle={
+        InteractiveTitle
+          ? provided => <InteractiveTitle {...provided.widgetData?.chart} />
+          : null
+      }
+      EmptyComponent={WidgetEmptyStateWarning}
+      Queries={{
+        // The queries aren't actually happening here, but we propagate these to manage loading state
```

Comment:
Yeah these were all built with RQ in mind (RQ query is supposed to replace these components) but I wasn't sure they'd be sticking around with Starfish so didn't port them.